### PR TITLE
Started a new branch and committed all of old changes. Added Hello World Response.

### PR DESCRIPTION
**Pull request by piercedwilson**
Hi Lu and Susan,
After reviewing the Git tutorial, I realized where I was misunderstanding, and I thought it would be easier to just begin a new branch with everything   and then do the rest of my changes for week 3, 4 and 5 here. Is that okay?

For this pull request I just added the first response on the index.html, DataServelet.java, and script.js

### CODE REVIEW

File: pierce-portfolio/portfolio/src/main/webapp/resume.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <div class="topnav">
+    <a href="/">About me</a>
+    <a class="active" href="/resume.html">Resume</a>
+    <a href="/fotos.html">Photos!</a>
+  </div>
+    <br> <br>
+    <h1> Resume </h1>
+    <p>Hi. On this page you can view my resume.</p>
+    <iframe src="https://docs.google.com/document/d/e/2PACX-1vRWpNZ2obmCnrbkWWm9-HSehUaBOkpiqxefZBjOBrSQtMR47eSmlyw06GkRaPasQBjhFct1HrPmGuRy/pub?embedded=true" width="1000" height = "1000"></iframe>
+
```
Comment (lupengscu) [inline pierce-portfolio/portfolio/src/main/webapp/resume.html]: Add closing tags

File: pierce-portfolio/portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+// limitations under the License.
+
+
+
+/**
+ * Adds a random greeting to the page.
+ */
+function addRandomGreeting() {
+  const greetings =
+      ['tomatoes are my favorite food', 'I ate an apple every day in 2019', 'I am a middle child', 'I am learning to drive right now', 'I went to boarding school for high school', 'I play the saxophone and piano'];
```
Comment (lupengscu) [inline pierce-portfolio/portfolio/src/main/webapp/script.js]: Consider divide the array into multiple lines

**Review by lupengscu — CHANGES_REQUESTED** [review]
Can you add a .gitignore file to exclude auto-generated files in target directory?

File: pierce-portfolio/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+// Did not remove this because it will be used in week 3
```
Comment (sliang3) [inline pierce-portfolio/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Can you switch this to "TODO(piercedw): Implement in week 3" so it's more obvious that these are action items?

File: pierce-portfolio/portfolio/src/main/webapp/fotos.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  </head>
+
+  <div class="topnav">
+    <a href="/">About me</a>
+    <a href="/resume.html">Resume</a>
+    <a class="active" href="/fotos.html">Photos!</a>
+
+  <body>
+    <div id="content">
+        <br><br>
```
Comment (sliang3) [inline pierce-portfolio/portfolio/src/main/webapp/fotos.html]: nit: fix spacing

File: pierce-portfolio/portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#greeting-container {
+  margin-top: 20px;
+}
+
+.image-1 {
```
Comment (sliang3) [inline pierce-portfolio/portfolio/src/main/webapp/style.css]: nit: image-1 isn't very descriptive, maybe we could do .photo or .image-photo?

File: pierce-portfolio/portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    height: 500px;
+    width: 600px;
```
Comment (sliang3) [inline pierce-portfolio/portfolio/src/main/webapp/style.css]: Also looks like there's some extra spaces here.

File: pierce-portfolio/portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  color: white;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+
```
Comment (sliang3) [inline pierce-portfolio/portfolio/src/main/webapp/style.css]: nit: Remove extra line

**Review by sliang3 — APPROVED** [review]

**Review by lupengscu — APPROVED** [review]

**Pull request merged**